enclave/src/polkadex_gateway.rs

```diff
@@ -581,6 +581,7 @@ pub fn consume_order(
 ) -> Result<(), GatewayError> {
```

Comment:
It would be better to rename current_order to taker and counter_order to maker, it makes more sense.